src/app/manager-dashboard/manager-dashboard.component.ts

```diff
@@ -1,12 +1,26 @@
-import { Component } from '@angular/core';
+import { Component, OnInit } from '@angular/core';
+import { UserService } from '../shared/user.service';
 
 @Component({
   template: `
-    <div><a routerLink="/community" i18n mat-raised-button>Communities</a> <a routerLink="/nation" i18n mat-raised-button>Nations</a></div>
+    <div *ngIf="displayDashboard"><a routerLink="/community" i18n mat-raised-button>Communities</a> <a routerLink="/nation" i18n mat-raised-button>Nations</a></div>
+    <div>{{message}}</div>
   `
 })
-export class ManagerDashboardComponent {
+export class ManagerDashboardComponent implements OnInit {
+  isUserAdmin = false;
+  displayDashboard = true;
+  message = '';
 
-  constructor() { }
+  constructor( private userService: UserService) { }
+
+  ngOnInit() {
+    Object.assign(this, this.userService.get());
```

Comment:
I think I encouraged something not so good by using this earlier.  Now that we have many properties from `this.userService.get()`, we shouldn't apply all of them to our classes.  Instead, you should simply assign `this.isUserAdmin  = this.userService.get().isUserAdmin`